### PR TITLE
Update storage account name in Azure registry storage configuration

### PR DESCRIPTION
**Pull request by bmcelvee**
Per Slack discussion.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@dmage would you mind reviewing this change, please? Thanks!

File: modules/registry-configuring-storage-azure-user-infra.adoc [hunk 1]
```diff
@@ -27,6 +27,6 @@ $ oc edit configs.imageregistry.operator.openshift.io/cluster
 
 storage:
   azure:
-    accountName: <account-name>
+    storage_account_name: <account-name>
```
Comment (dmage) [inline modules/registry-configuring-storage-azure-user-infra.adoc:30]:
```suggestion
    accountName: <storage-account-name>
```

Comment (bmcelvee) [reply modules/registry-configuring-storage-azure-user-infra.adoc]: Thank you! I thought that looked odd initially. :)

**Comment by bmcelvee** [comment]
No QE review necessary.

**Comment by codyhoag** [comment]
The change LGTM! :tada:

**Pull request merged**